knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -146,25 +203,25 @@ func (r *ReconcileKnativeKafka) ensureFinalizers(instance *operatorv1alpha1.Knat
 
 // Install Knative Kafka components
 func (r *ReconcileKnativeKafka) installKnativeKafka(instance *operatorv1alpha1.KnativeKafka) error {
-	if err := applyKnativeKafka(instance, r.client); err != nil {
+	if err := r.applyKnativeKafka(instance, r.client); err != nil {
```

Comment:
I think now the only remaining client usage is in the checkDeployments functions. Should we just pass r.client down in `installKnativeKafkaChannel/Source` and drop the passing through from everywhere?

Reply:
Done.